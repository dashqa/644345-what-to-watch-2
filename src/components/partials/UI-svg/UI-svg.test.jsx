import React from 'react';
import renderer from 'react-test-renderer';
import UISvg from './UI-svg';

const mock = {
  xlink: ``,
  viewBox: ``,
  width: 0,
  height: 0,
};

it(`UI svg correctly renders after relaunch`, () => {
  const {xlink, viewBox, width, height} = mock;
  const tree = renderer.create(
      <UISvg
        xlink={xlink}
        viewBox={viewBox}
        width={width}
        height={height}
      />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
