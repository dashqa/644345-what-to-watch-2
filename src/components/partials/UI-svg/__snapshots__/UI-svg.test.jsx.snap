// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI svg correctly renders after relaunch 1`] = `
<svg
  height={0}
  viewBox=""
  width={0}
>
  <use
    xlinkHref=""
  />
</svg>
`;
